ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/_cq_dialog/.content.xml

```diff
@@ -21,7 +21,7 @@
     jcr:title="Accordion"
     sling:resourceType="cq/gui/components/authoring/dialog"
     extraClientlibs="[core.wcm.components.commons.editor.dialog.childreneditor.v1,core.wcm.components.accordion.v1.editor,core.forms.components.panelcontainer.v1.editor]"
-    helpPath="https://www.adobe.com/go/aem_cmp_accordion_v1"
+    helpPath="https://www.adobe.com/go/aem_af_cmp_accordian_v1"
```

Comment:
doc team should correct the spelling of this goto url `aem_af_cmp_accordion_v1`

Reply:
Change the URL to https://www.adobe.com/go/aem_af_cmp_accordion_v1 in this PR, and update doc team